test/NuGet.Core.Tests/NuGet.Frameworks.Test/NuGetFrameworkParseTests.cs

```diff
@@ -228,6 +228,7 @@ public void NuGetFramework_ProfileName(string folder, string expected)
         [InlineData("netcoreapp3.0", ".NetCoreApp,Version=v3.0")]
         [InlineData("net5.0-android", "net5.0-android")]
         [InlineData("net5.0-android", "net5.0-android0.0")]
+        [InlineData("net5.0-android10.0", "net5.0-android10")]
```

Comment:
This test looks weird to me. 
Especially when comapred to the below 3.

Reply:
can you elaborate? :)

Reply:
It doesn't seem to round trip. 

`net5.0-android10.0` to `net5.0-android10` in this test, while the below test cases do preserve the `0`. 

In `        [InlineData("net5.0-ios14.0", "net5.0-ios14.0")]`, it's not removing the trailing 0.

Reply:
you're reading it the wrong way around, I think. The one on the right is the input, and the left is the expected

Reply:
You're reading it the other way around, I think? The expected output is on the left, and the second item is the input. Which... yes, is kinda confusing? It's not -supposed- to drop a .0

Reply:
Yeah I was reading it the other way around. 

I know xunit asserts have the `expected,actual` ordering, but when passing in inline parametrs, the ordering is *usually*, `input1, input2, output`. 

It's a crazy world :D